website/docs/reference/events-logging.md

```diff
@@ -34,7 +34,7 @@ _For more details about how the eventing system has been implemented in dbt-core
 
 <VersionBlock firstVersion="1.4">
 
-Starting in v1.4, the structure of each event in `dbt-core` is backed by a schema defined using [protocol buffers](https://developers.google.com/protocol-buffers). All schemas are defined in the [`types.proto`](https://github.com/dbt-labs/dbt-core/blob/main/core/dbt/events/types.proto) file within the `dbt-core` codebase.
+Starting in v1.4, the structure of each event in `dbt-core` is backed by a schema defined using [protocol buffers](https://developers.google.com/protocol-buffers). All schemas are defined in the [`types.proto`](https://github.com/dbt-labs/dbt-core/blob/main/core/dbt/events/core_types.proto) file within the `dbt-core` codebase.
```

Comment:
i think given the context, this is the right link we want to link to right @jtcohen6 ? there's another 'types.proto' file in the `dbt-adapter` repo but it seem like this is the more appropriate one?